Tests/Tests/Core/Utilities/AttributedStringBuilder/AttributedStringBuilderTests.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  AttributedStringBuilderTests.swift
+//  PovioKit_Tests
+//
+//  Created by Ndriqim Nagavci on 27/10/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import XCTest
+@testable import PovioKit
+
+class AttributedStringBuilderTests: XCTestCase {
+  let text = "My custom text!"
+  let substring = "custom"
+  let textColor: UIColor = .red
+  let font: UIFont = .boldSystemFont(ofSize: 14)
+  let underlineStyle: NSUnderlineStyle = .thick
+  let paragraphLineSpacing: CGFloat = 10
+  let paragraphLineHeight: CGFloat = 30
```

Comment:
Would it be better if we move all these helper properties below in an extension?
e.g.
`MARK: - Helpers
extension AttributedStringBuilderTests {
  func anyText() -> String {
    "My custom text!"
  }
}

Reply:
Maybe we can use enum in this case?
`// MARK: - Helpers
private extension AttributedStringBuilderTests {
  enum Values {
    static let text = "My custom text!"
  }
}

Thoughts?

Reply:
Looks good.